vendor/scripts/register-game-libraries.js

```diff
@@ -0,0 +1,3 @@
+require.register("game-libraries", function(exports, require, module) {
+	exports = {createjs: window.createjs};
```

Comment:
This line doesn't do anything, and I think this file isn't needed since the ModuleLoader is returning {} when 'game-libraries' is required.